cluster.go

```diff
@@ -166,3 +177,13 @@ func destroyCluster(c *Cluster) error {
 
 	return deleteVMs(vmNames)
 }
+
+func extendCluster(c *Cluster, extension time.Duration) error {
+	newLifetime := c.Lifetime + extension
+	for _, vm := range c.VMs {
+		if err := extendVM(vm.Name, newLifetime); err != nil {
```

Comment:
Does it help to do this in parallel? For example:

```
  var wg sync.WaitGroup
  wg.Add(len(c.VMs))
  for i := range c.VMs {
    go func(i int) {
      defer wg.Done()
      vm := c.VMs[i]
      ...
    }(i)
  }
  wg.Wait()
```

Reply:
Sude, done.